VSConfigFinder/CommandLineOptions.cs

```diff
@@ -1,18 +1,19 @@
 ﻿namespace VSConfigFinder
 {
-    internal class CommandLineOptions : ICommandLineOptions
-    {
-        public CommandLineOptions(string filePath, string output)
-        {
-            Utilities.ValidateIsNotNullOrEmpty(filePath, nameof(filePath));
-            Utilities.ValidateOutputParameter(output, nameof(output));
-
-            FilePath = filePath;
-            Output = output;
-        }
+    using CommandLine;
 
-        public string FilePath { get; set; }
+    /// <inheritdoc/>
+    public class CommandLineOptions : ICommandLineOptions
+    {
+        /// <inheritdoc/>
+        [Option("folderpath", Required = true, HelpText = "Set the source folder path to use as the root. The search will start from the root towards the bottom.")]
+        public string FolderPath { get; set; }
 
-        public string Output { get; set; }
+        /// <inheritdoc/>
+        [Option("createfile", Required = false, HelpText = "Set the folder path to output the consolidated .vsconfig.\n" +
+            "If the argument (--createfile) is not passed at all, the program will output the cli arguments that can be fed into the Visual Studio Installer setup arguments. (e.g. --add --add Microsoft.VisualStudio.Component.Roslyn.Compiler --add Microsoft.Net.Component.4.8.SDK\n" +
```

Comment:
1. Is there supposed to be two consecutive "--add" in the example?
2. When I read the [msft doc](https://learn.microsoft.com/en-us/visualstudio/install/use-command-line-parameters-to-install-visual-studio?view=vs-2022#:~:text=on%20this%20page.-,%2D%2Dadd%20%3Cone%20or%20more%20workload%20or%20component%20IDs%3E,-Optional%3A%20One%20or), I thought we can pass in multiple ids with one "--add". Am I missing something?
3. I think it's missing a closing parentheses at the end.

nit: I usually say "argument is passed **in**", but not sure if that's the proper English haha

Reply:
1. Yes.
2. Also yes. Both make sense. We can update it so that we only output "--add" one time.
3. Thanks! Will update.